### DISCUSSION TITLE
Building pip package for Windows 10

### DISCUSSION THREAD

**Discussion by ghost** [tensorflow/compression#79] [General]
Hello, 
I'm using Windows10 and my versions are tensorflow-gpu=2.0.0, python=3.7 and tensorflow_compression=2.1 
Since, I want to train and use your models with my host GPU, I tried to build my own pip package with Bazel (0.26.1)
At the web page "https://github.com/tensorflow/custom-op" , it is said that skipping Docker steps, running configure file and build with bazel in the root directory, for Windows users. 
When I did all these, and after "bazel build-c opt --copt=-mavx :build_pip_pkg" command, I encountered the following error "An error occured during the fetch of repository 'tensorflow_pip''. I'm not familiar with the whole building custom ops. Please, can you summarize the steps for locally building this pip package at Windows 10 ?

Thank you in advance for your help.

**Comment by jonaballe** [comment]
Hi, unfortunately we have no experience with building Windows packages either. However, if you find out a way to do it, we would consider updating the documentation to provide guidance for doing this.

On the web site you linked, it says you need a similar setup as discussed here: https://www.tensorflow.org/install/source_windows. Are these the instructions you have been following?

**Reply by ghost** [reply]
Yes, I followed these instructions but before building pip package, there must be a configuration file. Since there was no configuration file in the  code directiory, I actually skipped that part.

Thank you for your answer.
